stock_quantity_history_location/tests/test_stock_quantity_history_location.py

```diff
@@ -31,35 +31,43 @@ def setUpClass(cls):
         )
         cls._create_stock_move(cls, location_dest_id=cls.child_test_stock_loc, qty=100)
 
-    def test_01_wizard_past_date(self):
+    def get_stock_quantity_history_action(self, location, to_date):
         wizard = self.env["stock.quantity.history"].create(
             {
-                "location_id": self.test_stock_loc.id,
-                "include_child_locations": True,
-                "inventory_datetime": fields.Datetime.now(),
+                "location_id": location.id,
+                "inventory_datetime": to_date,
             }
         )
-        action = wizard.with_context(company_owned=True).open_at_date()
+        return wizard.open_at_date()
+
+    def test_stock_quantity_history_location(self):
+        current_date = fields.Datetime.now()
+        past_date = fields.Datetime.to_datetime("2019-08-10 00:00:00")
+        action = self.get_stock_quantity_history_action(
+            self.child_test_stock_loc, current_date
+        )
+        to_date_in_context = action["context"].get("to_date")
+        self.assertIsNotNone(to_date_in_context)
+        self.assertEqual(to_date_in_context, current_date)
```

Comment:
Non-blocking. I think this should be sufficient.
```suggestion
        self.assertEqual(action["context"].get("to_date"), current_date)
```